--- KvArcDesktopMenuKwin/k	2019-09-14 11:02:16.534925100 +0800
+++ KvArcDesktopMenuKwin/KvArcDesktopMenuKwin.kvconfig	2019-09-14 11:02:11.612465230 +0800
@@ -11,7 +11,7 @@
 toolbar_interior_spacing=3
 spread_progressbar=true
 composite=true
-menu_shadow_depth=7
+menu_shadow_depth=7
 menu_separator_height=6
 tooltip_shadow_depth=6
 splitter_width=4
@@ -30,12 +30,12 @@
 double_click=false
 translucent_windows=false
 blurring=false
-popup_blurring=false
+popup_blurring=true
 vertical_spin_indicators=false
 spin_button_width=32
 fill_rubberband=false
 merge_menubar_with_toolbar=true
-small_icon_size=16
+small_icon_size=32
 large_icon_size=32
 button_icon_size=16
 toolbar_icon_size=22
